File: week-04/day-01/Sharpie.py
class Sharpie(object):
    def __init__(self, color, width, ink_amount = 100.0):
        self.color = str(color)
        self.width = float(width)
        self.ink_amount = ink_amount

    def use(self):
        self.ink_amount -= 1

new_sharpie = Sharpie("pink", 10)
new_sharpie.use()
print(new_sharpie.ink_amount)

File: week-08/Sharpie.js
class Sharpie {
  constructor(color, width) {
    this.color = color;
    this.width = width;
    this.inkAmount = 100;
  }

  use() {
    if (this.inkAmount < this.width) {
      this.width = this.inkAmount;
      this.inkAmount -= this.width;
    } else {
      this.inkAmount -= this.width;
    }
  }
}

let sharpieOne = new Sharpie('pink', '101');
let sharpieTwo = new Sharpie('blue', '1');

console.log(sharpieOne);
console.log(sharpieTwo);

sharpieOne.use();
sharpieTwo.use();

console.log(sharpieOne);
console.log(sharpieTwo);